hls4ml/templates/vivado/nnet_utils/nnet_recurrent.h

```diff
@@ -235,6 +234,41 @@ void lstm_stack(data_T data[CONFIG_T::n_sequence * CONFIG_T::n_in], res_T res[CO
         }
 }
 
+template <class data_T, class data2_T, class data3_T, class res_T, typename CONFIG_T>
```

Comment:
Any more descriptive names than `data2_T` and `data3_T`?

Reply:
I tried to put more descriptive names in the oneAPI version. Maybe we can go more in that direction for the other backends (and potentially standardize).